net-im/signal-cli-bin/signal-cli-bin-0.11.5.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="signal-cli-${PV}"
+DESCRIPTION="Send and receive messages of Signal Messenger over a command line interface"
+HOMEPAGE="https://github.com/AsamK/signal-cli/wiki"
+SRC_URI="
+	https://github.com/AsamK/signal-cli/releases/download/v${PV}/${MY_P}-Linux.tar.gz -> ${P}.tar.gz
+	https://github.com/AsamK/signal-cli/raw/v${PV}/README.md -> ${P}.README.md
```

Comment:
I'm not sure how much useful can be download the README as we normally delete it